core/src/main/scala/kafka/tools/ConsumerPerformance.scala

```diff
@@ -158,8 +158,22 @@ object ConsumerPerformance {
     val elapsedMs: Double = endMs - startMs
     val totalMBRead = (bytesRead * 1.0) / (1024 * 1024)
     val mbRead = ((bytesRead - lastBytesRead) * 1.0) / (1024 * 1024)
+    
```

Comment:
@vahidhashemian This code is written in late 2011 and developers at that time probably doesn't spend much time on such issue in tools and instead focus on implementing new features, since they are probably the only users of such tools. Recent committers are enforcing much high standards and they will probably ask such questions when reviewing your code. I myself have been forced to make my patch bullet-proof to any question like this before submitting it for review.